lib/protoparser/prometheus/parser.go

```diff
@@ -147,16 +147,20 @@ func (r *Row) unmarshal(s string, tagsPool []Tag, noEscapes bool) ([]Tag, error)
 	return tagsPool, nil
 }
 
+var rowsReadScrape = metrics.NewCounter(`vm_protoparser_rows_read_total{type="promscrape"}`)
```

Comment:
Could you share the purpose of this metric?

Probably the `vm_protoparser_rows_read_total{type="prometheus"}` metric at https://github.com/VictoriaMetrics/VictoriaMetrics/blob/e5b9f47623fe332fdbe7e525101d87d68a0515c6/lib/protoparser/prometheus/streamparser.go#L109 should be enough? This metric is updated when `prometheus.ParseStream()` is called.

Reply:
Nope. Streamparser are not used in scraping.

Reply:
The main purpose is to add Rows from scraping.

Reply:
Agreed.